conftest.py

```diff
@@ -157,6 +157,21 @@ def side_effect(args, *other_args, **kwargs):
         yield
 
 
+@pytest.fixture(autouse=True)
```

Comment:
As this is defined in the root `conftest.py`, it will also be auto-used under `tests/integration_tests`. Is that intentional? If not, we could move this definition to `tests/unittests/conftest.py`.

Reply:
I didn't check, but I would assume this isn't necessary for integration tests. I'll update it accordingly. Thanks!